pages/1.11/networking/DNS/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: layout.pug
+navigationTitle:  DNS
+title: DNS Quick Reference
+menuWeight: 20
+excerpt:
+
+enterprise: false
+---
+
+<!-- This source repo for this topic is https://github.com/dcos/dcos-docs -->
+DC/OS provides a highly distributed and faulterant DNS-based service discovery mechanism. DNS is provided by two different components within DC/OS, [mesos-dns](/pages/1.11/networking/DNS/mesos-dns) and [dcos-dns](/pages/1.11/networking/DNS/dcos-dns). DNS service in DC/OS supports two  top-level domain names (TLD) `.mesos` and `.directory`. All records for the `.mesos` TLD are served by `mesos-dns` and all records for `.directory` TLD are served by `dcos-dns`. Please read the "Recommendation" section to better understand the usage of these two TLDs. Each of the TLD consist of multiple zones. Every DC/OS service gets multiple FQDN entries, from these different zones. Below we describe the different zones and the FQDN associated with those zones and the type of network connectivity that a service recieves when it is accessed using an FQDN from a particular zone.
+
+For the purpose of illustration lets assume that we are running the following Docker container on DC/OS:
+```json
+{
+  "id": "/mygroup/myapp",
+  "instances": 1,
+  "container": {
+    "type": "DOCKER",
+    "volumes": [],
+    "docker": {
+      "image": "nginx"
+    },
+    "portMappings": [
+      {
+        "containerPort": 80,
+        "labels": {
+          "VIP_0": "/mygroup/myapp:80"
+        },
+        "name": "http"
+      }
+    ]
+  },
+  "cpus": 0.1,
+  "mem": 128,
+  "requirePorts": false,
+  "networks": [
+    {
+      "name": "dcos",
+      "mode": "container"
+    }
+  ],
+  "healthChecks": [],
+  "fetch": [],
+  "constraints": []
+}
+```
+The `/mygroup/myapp` is running on a virtual network called "dcos". We will use this app to describe the different FQDN it receives from DC/OS DNS.
+
+*NOTE*: While the example uses a Docker container the same FQDNs are generated for a UCR container as well.
+
+# `myapp.mygroup.marathon.mesos`
+For any service launched on DC/OS, every instance of the service would get the FQDN `<service-name>.<group-name>.marathon.mesos`. 
+
+This FQDN is resolved by `mesos-dns`. Please read the "Recommendation" section to understand the difference between using this FQDN vs using an FQDN from the `*.directory` TLD. The actual IP address to which this FQDN maps will depend on the networking mode on which the container is running. For our given example, since the application is running on an SDN the IP address will be the container's IP address. That said, if the application was running on host mode networking, than the IP address would have been that of the agent on which the application instance is running. For bridge mode networking, the IP address would again be that of the agent since the application would be accessible only through D-NAT rules on the agent.
```

Comment:
s/That said, if the application was running on host mode networking, than the IP address would have been that of the agent on which the application instance is running./If the application was running on host mode networking, the IP address would have been that of the agent on which the application instance is running./